src/server/qgsfeaturefilterprovidergroup.h

```diff
@@ -0,0 +1,61 @@
+/***************************************************************************
+                       qgsfeaturefilterprovidergroup.h
+                       -------------------------------
+  begin                : 26-10-2017
+  copyright            : (C) 2017 by Patrick Valsecchi
+  email                : patrick dot valsecchi at camptocamp dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSFEATUREFILTERPROVIDERGROUP_H
+#define QGSFEATUREFILTERPROVIDERGROUP_H
+
+#include "qgsfeaturefilterprovider.h"
+#include "qgis_server.h"
+
+#include <QList>
+
+/**
+ * \ingroup server
+ * \class QgsAccessControl
```

Comment:
Same. 
A general question (sorry if I didn't really read all the code): is this a replacement of the accesscontrol filters (and plugins) system that @sbrunner  developed a few years ago ?

Reply:
Not at all. I'm just using the infa put in place for that to add additional filters.